src/status_im/extensions/ethereum.cljs

```diff
@@ -204,3 +206,25 @@
            registry (get ens/ens-registries chain)]
        (ens/get-addr web3 registry name #(re-frame/dispatch (on-result {:result %}))))
      (re-frame/dispatch (on-result {:error (str "'" name "' is not a valid name")})))))
+
+;; EXTENSION SIGN -> SIGN MESSAGE
+(defn  execute-extension-sign [db {:keys [address message data id on-result]}]
```

Comment:
What happens if the address is not the current wallet one?

Reply:
<img width="372" alt="schermata 2018-11-30 alle 08 02 27" src="https://user-images.githubusercontent.com/15999009/49273767-58ec4e80-f476-11e8-93e9-c30b0c42707d.png">

I think makes sense to remove address parameter, unless in some cases we have more then one account unlocked